types/testing-library__dom/events.d.ts

```diff
@@ -80,7 +80,8 @@ export type EventType =
     | 'pointerOut'
     | 'pointerLeave'
     | 'gotPointerCapture'
-    | 'lostPointerCapture';
+    | 'lostPointerCapture'
+    | 'popstate';
```

Comment:
Hmmm... Shouldn't this be `popState`?

Reply:
I had a look earlier but https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onpopstate seems to suggest it is `popstate`?

Reply:
Also they seem to  use `popstate` in JSDOM [1] so I think it's correct?

[1] https://github.com/jsdom/jsdom/blob/d6f8a97b5fb7709d6ad0215c1ae95fd4cab58489/lib/jsdom/living/nodes/WindowEventHandlers-impl.js#L18

Reply:
Wait, there is also this PR for `dom-testing-library` which uses `popState`: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41951/files 

I am confused now

Reply:
[It is `popState`](https://github.com/testing-library/dom-testing-library/pull/433/files#diff-4607e46b15da8ae44c4b47730c6d5963R342)

Reply:
Thanks I will cancel this PR since there is another one. And I got it wrong it should be popState